<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Admin: SUNLAB Card Swipe Access Database</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f4f7; color: #333; font-family: "Roboto", sans-serif;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f0f4f7; color: #333;</string>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Access Log</string>
     </attribute>
     <widget class="QLineEdit" name="IDSearchTextBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>350</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff; border: 1px solid #ccc; border-radius: 5px; font-size: 12pt; padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>ID Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="filterLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 12pt; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Filter:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="studentTableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>350</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff; border: 1px solid #ddd; font-size: 11pt;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshTableButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>440</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1abc9c; color: white; border-radius: 10px; font-size: 12pt; font-weight: bold; padding: 5px 20px;</string>
      </property>
      <property name="text">
       <string>Refresh Tables</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="dateSelect">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>200</y>
        <width>300</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff; border: 1px solid #ccc; font-size: 12pt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeData">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>420</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff; border: 1px solid #ccc; border-radius: 5px; font-size: 12pt; padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>After HH:MM:SS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="allDayRadio">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>390</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 12pt; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>All Day?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Access Control</string>
     </attribute>
     <widget class="QTableWidget" name="adminTableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>550</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff; border: 1px solid #ddd; font-size: 11pt;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="IDSearchTextBox2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff; border: 1px solid #ccc; border-radius: 5px; font-size: 12pt; padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>ID Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="giveAccess">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>160</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1abc9c; color: white; border-radius: 10px; font-size: 12pt; font-weight: bold; padding: 5px 20px;</string>
      </property>
      <property name="text">
       <string>Give Access</string>
      </property>
     </widget>
     <widget class="QPushButton" name="suspendAccess">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e74c3c; color: white; border-radius: 10px; font-size: 12pt; font-weight: bold; padding: 5px 20px;</string>
      </property>
      <property name="text">
       <string>Suspend Access</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>220</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="title">
       <string>Messages</string>
      </property>
      <widget class="QLabel" name="messageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>130</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2ecc71; font-size: 12pt; font-weight: bold; text-align: center;</string>
       </property>
       <property name="text">
        <string>Access Updated</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
